### ISSUE TITLE
Split atinux or master branch to another repo?

### ISSUE THREAD

**Issue by detrohutt** [electronuxt/electronuxt-starter#4]
@Atinux I checked out your code and it's brilliant; much, much simpler than mine and works great, even on Windows. I'm definitely going to use your code. However, I had something specific in mind for the current code I have.. I want people to eventually be able to combine any nuxt vue-cli template (even future ones) with electron with basically one command: `npm start [<template-name>]` or something simliar. 

So I'm thinking I'd eventually package vue-cli with the current code and add config options for adjusting start commands for different templates, things like that. So I'm considering migrating what I've got so far to another repo, maybe called 'electronuxt-cli' or something. Then I could just replace the master branch of this repo with your code, because your code fits the 'starter' name better.

What do you think?

**Comment by wormen** [comment]
@detrohutt I suggest to move to the repository **nuxt/electron**

**Comment by detrohutt** [comment]
@wormen Good suggestion. I'm fine with doing that if @atinux wants to. My reason for creating this organization was so they wouldn't have to support/maintain electron repos on top of all the other repos they're having to maintain already.

**Comment by Atinux** [comment]
I can create the repo and give you full access @detrohutt

**Comment by detrohutt** [comment]
Thanks @Atinux! I'm on my way to bed but when I get up I'll move the atinux branch of this repo over to the new repo.

**Comment by Atinux** [comment]
I did it, so you can start directly from there :)

I was thinking that it could be something that wrap a nuxt app, like we
have a folder inside with a nuxt app, we can set the folder name so
electronuxt can launch it?

**Comment by detrohutt** [comment]
Yeah that would be cool! Electron also handles drag 'n' drop so maybe we could also give them the option of just running the app and if the folder isn't set they can drop the folder into the electron window, maybe have a browse button too?

**Comment by Atinux** [comment]
This is a good question @detrohutt 

I don't really know since it will clone the folder I guess.
I think it's better to have a small template where you can package your app and run it, keep it simple :)
